backend/src/test/java/com/htecgroup/skynest/utils/FolderEntityUtil.java

```diff
@@ -14,6 +17,7 @@ public static FolderEntity getFolderWithoutParent() {
     FolderEntity folderEntity = new FolderEntity(null, bucketEntityUtil);
     folderEntity.setName("FolderName");
     folderEntity.setCreatedBy(UserEntityUtil.getVerified());
+    folderEntity.setId(UUID.randomUUID());
```

Comment:
Try to set exact id so that tests which compares that folder id with another one have easier setup. Use UUID.fromString and add some random generated uuid value which will be always set when you call this method. We need to exclude these random id fields from test util classes for later use.

Reply:
Actually id is not needed here at all, I was trying something and forgot to remove it, I will take it out, but when we need id in the future we'll do it like you say.